### PR TITLE
Upgrade to ember-cli 2.13

### PR DESCRIPTION
**Pull request by ahmadsoe**
Currently, ember app using `ember-cli-babel@6` will break when installing this addon. 

> ember-cli-htmlbars-inline-precompile@0.3 requires the host to use ember-cli-babel@5. To use ember-cli-babel@6 please upgrade ember-cli-htmlbars-inline-precompile to 0.4.

This PR will also resolve that issue.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -2,11 +2,8 @@ machine:
   node:
     version: "stable"
 dependencies:
-  pre:
```
Comment (poteto) [inline circle.yml]: Could you please update this file to:

```yml
machine:
  node:
    version: "stable"
  environment:
    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"

dependencies:
  override:
    - yarn
  cache_directories:
    - ~/.cache/yarn

test:
  override:
    - yarn test
```

File: package.json [hunk 2]
```diff
@@ -2,56 +2,57 @@
...
     "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-export-application-global": "^1.0.5",
-    "ember-load-initializers": "^0.5.1",
-    "ember-resolver": "^2.0.3",
-    "loader.js": "^4.0.11",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^4.0.0",
+    "ember-source": "~2.13.0",
+    "ember-welcome-page": "^3.0.0",
```
Comment (poteto) [inline package.json]: Please remove this

File: testem.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
-/*jshint node:true*/
+/* eslint-env node */
 module.exports = {
-  "framework": "qunit",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-    "Chrome"
+    "PhantomJS"
```
Comment (poteto) [inline testem.js]: Please change this back to `Chrome`

File: testem.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
   ],
   "launch_in_dev": [
+    "PhantomJS",
```
Comment (poteto) [inline testem.js]: Please remove `PhantomJS`

File: tests/dummy/app/templates/application.hbs [hunk 5]
```diff
@@ -1,3 +1,5 @@
-<h2 id="title">Welcome to Ember</h2>
+{{!-- The following component displays Ember's default welcome message. --}}
+{{welcome-page}}
```
Comment (poteto) [inline tests/dummy/app/templates/application.hbs]: Please undo the change to this file.

**Comment by ahmadsoe** [comment]
@poteto - all done, should be good to go.

**Comment by musaffa** [comment]
@poteto Please release a new version.

**Comment by poteto** [comment]
Released in `1.2.7`
